include/spblas/algorithms/multiply_impl.hpp

```diff
@@ -20,6 +20,10 @@ void multiply(A&& a, B&& b, C&& c) {
         "multiply: matrix and vector dimensions are incompatible.");
   }
 
+  for (std::size_t i = 0; i < __backend::shape(c)[0]; i++) {
```

Comment:
`__backend::shape(c)` when `c` is a vector returns an integer value, so this should just be `__backend::shape(c)`.